target/linux/ath79/dts/qca9563_nec_wg800hp.dts

```diff
@@ -142,6 +142,24 @@
 					#address-cells = <1>;
 					#size-cells = <1>;
 
+					macaddr_board_data_280: macaddr@280 {
+						compatible = "mac-base";
+						reg = <0x280 0x11>;
+						#nvmem-cell-cells = <1>;
+					};
+
+					macaddr_board_data_480: macaddr@480 {
+						compatible = "mac-base";
```

Comment:
This cell is unused.

Reply:
This is here for the future when DSA comes. Right now 02_network has

wan_mac=$(mtd_get_mac_text board_data 0x480)

for this. There's only one ethernet interface.